defaults/main.yml

```
@@ -30,3 +30,6 @@ clamav_scan_quarantine_directory: /var/spool/clamav
 clamav_scan_quarantine_group: root
 clamav_scan_quarantine_mode: 0750
 clamav_scan_quarantine_owner: root
+
+clamav_seboolean_name: antivirus_can_scan_system
+clamav_seboolean_state: yes
```

I believe we prefer `true`/`false` to `yes`/`no` (and the [documentation for the module](https://docs.ansible.com/ansible/latest/collections/ansible/posix/seboolean_module.html#parameter-state) does as well).

```suggestion
clamav_seboolean_state: true
```

---

@mcdonnnj - If this is a preference of ours then I don't think I was aware of it.  I think I tend to use `yes` and `no` by default.  Is this documented somewhere?

---

@jsf9k It is the predominant thing I see in our Ansible configurations outside of the molecule configurations (which are duplicated out from a single source of truth). I don't know that we've codified it anywhere but it seems to be the general trend in practice. Specifically for this repo the other boolean default values use `true`/`false` so for consistency this one should as well.